pinot-core/src/main/java/com/linkedin/pinot/core/indexsegment/IndexSegment.java

```diff
@@ -17,7 +17,7 @@
 
 import com.linkedin.pinot.common.segment.SegmentMetadata;
 import com.linkedin.pinot.core.common.DataSource;
-import com.linkedin.pinot.core.startree.StarTree;
+import com.linkedin.pinot.core.startree.StarTreeInterf;
```

Comment:
I prefer name the interface StarTree, and name the implementation something else (maybe StarTreeImpl). Same as StarTreeNode.